fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -796,6 +801,43 @@ public void setBloomFilterInfo(Set<String> bfColumns, double bfFpp) {
         this.bfFpp = bfFpp;
     }
 
+    public void setSequenceInfo(Type type) {
+        this.hasSequenceCol = true;
+        this.sequenceType = type;
+
+        // sequence column is value column with MAX aggregate type
+        // this makes it easy to compare and update the sequence column
+        Column sequenceCol = new Column(Column.SEQUENCE_COL, type, false, AggregateType.MAX, false, null, "", false);
```

Comment:
add AggregateType.MAX to unique table ??

Reply:
I  have updated, sequence column use REPLACE

Reply:
I  have updated, sequence column use REPLACE